sass/_SmartSnippet.scss

```diff
@@ -0,0 +1,40 @@
+@import 'Variables';
+
+.CoveoSmartSnippet {
+  @include defaultMediumRoundedBorder();
+  margin-top: 30px;
+  padding: 16px;
+
+  display: none;
+  &.coveo-smart-snippet-has-answer {
+    display: block;
+  }
+
+  .coveo-smart-snippet-source {
+    margin-top: 24px;
+
+    .coveo-smart-snippet-source-url {
+      display: block;
+      color: $color-strong-contrast-grey;
+      text-decoration: none;
+
+      &:hover,
+      &:focus {
+        text-decoration: underline;
+      }
+    }
+
+    .coveo-smart-snippet-source-title {
```

Comment:
Those classnames are precise enough to decapsulate them a bit.
If you're a customer wanting to change the font-size here, you would need to write the whole same path:

.CoveoSmartSnippet .coveo-smart-snippet-source .coveo-smart-snippet-source-title {}

Which is a bit long, I'd try to keep them at 2 levels deep or under.

Reply:
Alright, I'm going to extract `.coveo-smart-snippet-source` out of `.CoveoSmartSnippet`.